src/prefect/logging/logging.yml

```diff
@@ -83,6 +125,12 @@ loggers:
     prefect.orion:
         level: "${PREFECT_LOGGING_SERVER_LEVEL}"
 
+    prefect.client:
+        level: "${PREFECT_LOGGING_SERVER_LEVEL}"
+
+    prefect.infrastructure:
+        level: "${PREFECT_LOGGING_SERVER_LEVEL}"
```

Comment:
These should not be SERVER level? These are client-side logs.

Reply:
I see. These were failing tests earlier because the tests before didn't account for two debug logs.